test/selectors/dashboard-selectors_spec.js

```diff
@@ -1,54 +1,58 @@
 import { expect } from 'chai'
 import { describe, it } from 'mocha'
 import { fromJS } from 'immutable'
-import { getSortedStudents, getStudentProgress } from '../../js/selectors/dashboard-selectors'
+import { getSortedStudents, getStudentProgress, getStudentTotalProgress } from '../../js/selectors/dashboard-selectors'
+import { SORT_BY_NAME, SORT_BY_MOST_PROGRESS, SORT_BY_LEAST_PROGRESS } from '../../js/actions/dashboard'
 
 describe('dashboard selectors', () => {
   const s1 = { id: 1, firstName: 'X', lastName: 'C' }
-  const s2 = { id: 2, firstName: 'y', lastName: 'b' }
+  const s2 = { id: 2, firstName: 'Y', lastName: 'AB' }
   const s3 = { id: 3, firstName: 'Z', lastName: 'A' }
```

Comment:
I've updated these names in my branch a bit to test your use case. Actually, I've used lower case on purpose to test if we ignore it. You can use them (or update these), as they test both lower vs uppercase and concatenation issue.